packages/configuration-validation/yarn.lock

```diff
@@ -2600,7 +2601,7 @@ lodash.sortby@^4.7.0:
   version "4.7.0"
   resolved "https://registry.yarnpkg.com/lodash.sortby/-/lodash.sortby-4.7.0.tgz#edd14c824e2cc9c1e0b0a1b42bb5210516a42438"
 
-lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.4, lodash@^4.2.0, lodash@^4.3.0:
+lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.4, lodash@^4.2.0, lodash@^4.3.0:
```

Comment:
Why does this still have 4.17.4, etc?  There are some notable security issues on those, do we have some other deps we should update the minimum version on?

Reply:
I'm not sure about this. I added the lodash dependency, so there should be nothing else that depends on it. I will clean and update yarn locks to see if this changes anything